cmd/juno/juno.go

```diff
@@ -202,6 +203,7 @@ func main() {
 	})
 
 	if err := cmd.ExecuteContext(ctx); err != nil {
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
I think this is redundant because cobra prints the error

Reply:
Thanks, forgot about it.